park_info.clj

```diff
@@ -1,198 +1,197 @@
-        :toilet true
-        :toilet_extra "in the restaurant and club"
-        :POIs "Historically significant sights, art gallery, musical club with restaurant in garden, playground for small children"
-        ;---Children and Sport---
-        :bike_path true
-        :skating false
-        :sports_field true
-        :sports_field_extra: "Tennis courts and sports facilities in Sokolovna gymnasium"
-        :playground true
-        :playground_extra "by the garden restaurant"
-        ;---Directions---
-        :public_transport "
-        stop: Klamovka - buses: #149, #191, #217, - trams: #4, #7, #9, #10, #58, #59"
-        :GPS_coords "50°4′17,04″ N, 14°22′39,72″ E"
-        :parking false
-        :parking_extra nil
-        ;---Additional Information---
-        :dogs true
-        :path_type "paved & asphalt"
-        :administered_by "Municipality Prague 5"
-        :opening_hours "Open year-round"
-        ;---Map---
-        :map_url "https://www.praha.eu/public/16/84/48/184296_4_klamovka_mapa.pdf"       
-       
-    }
-    def Ladronka{
+(def Obora_Hvezda{
+    ;---Basic Info---
+    :food true
+    :food_extra "During Summer Season on the premises of the Villa (www.pamatniknarodnihopisemnictvi.cz), tel:. +420 235 357 938,
+    or at Libocká street, restaurant U Holečků, www.uholecku.cz,
+    or at Libocká studna, tel. +420 777 836 966,
+    alternatively at Petřiny"
+    :toilet true
+    :toilet_extra "Portable toilets on the premises, during summer season toilets are also available in the villa"
+    :POIs "Suitable for tourists and families with children, heritage trees, nature trail, natural attractions"
+    ;---Children and Sport---
+    :bike_path true
+    :skating false
+    :sports_field false
+    :sports_field_extra "nearby: Youth House (www.ddmp6.cz/kontakt/) & traffic playground"
+    :playground true
+    :playground_extra "2 playgrounds"
+    ;---Directions---
+    :public_transport "
+    stop: Nad Markétou - buses: #179, #184, #191, #510, 
+    stop: Libocká - buses: #179, #510, 
+    stop: Vypich - trams: #15, #22, #25, #57"
+    :GPS_coords "50°5′2.798″ N, 14°19′45.529″ E"
+    :parking true
+    :parking_extra "8 parking spots at Libocká gate, free parking in U Hvězdy street"
+    ;---Additional Information---
+    :dogs false
+    :path_type "Hardened soil, threshing floor"
+    :administered_by "Museum of Czech Literature, Prague City"
+    :opening_hours "park - year long, villa - (April 10:00-17:00, May-September 10:00-18:00, October 10:00-17:00, November-March CLOSED) "
+    ;---Map---
+    :map_url "https://www.praha.eu/public/bf/26/3e/184361_4_hvezda_mapa.pdf"
+})
+
+(def Kampa{
+    ;---Basic Info---
+    :food true
+    :food_extra "restaurant: Rybí Restaurace,
+    café: Mlýnská (tel. +420 608 444 490)
+    restaurant/café : Sovovy Mlýny"
+    :toilet true
+    :toilet_extra nil
+    :POIs "View of the river, Sovovy Mlýny gallery, Čertovka, Charles Bridge, Werich House, heritage trees"
```

Comment:
No 'suitable for'. Either keep it everywhere, or cut it out everywhere I would think.

Reply:
These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.